features/admin.server-configurations.v1/constants/server-configurations-constants.ts

```diff
@@ -443,4 +443,14 @@ export class ServerConfigurationsConstants {
     public static readonly ORGANIZATION_SETTINGS_CATEGORY_ID: string = "organization-settings";
     public static readonly EMAIL_DOMAIN_DISCOVERY: string = "ZW1haWwtZG9tYWluLWRpc2NvdmVyeQ==";
     public static readonly IMPERSONATION: string = "impersonation";
+
+    /**
+     * Login Security Category Constants.
+     */
+    public static readonly SIFT_CONNECTOR_ID: string = "c2lmdC1jb25maWd1cmF0aW9u";
```

Comment:
There is LOGIN_ATTEMPT_SECURITY constant in L438 which falls under login security category constants. Shall we move it there or move that constant here for better readability?